src/taxpasta/infrastructure/application/__init__.py

```diff
@@ -61,6 +61,11 @@
     MotusProfileReader,
     MotusProfileStandardisationService,
 )
+from .ganon import (
+    GanonProfile,
+    GanonProfileReader,
+    GanonProfileStandardisationService,
+)
```

Comment:
I've been trying to keep the profilers in alphabetical order 😉

Reply:
Will fix everywhere! didn't notice that sorry!